ConwaysGameOfLife/Generator.cs

```diff
@@ -28,13 +32,39 @@ public Generator()
 
         */
 
-        public bool ShouldSwitchCellState()
+        public Universe GenerateNewUniverse()
+        {
+            var oldCells = _oldUniverse.Cells;
+            var newCells = new List<Cell>();

```

Comment:
move all code pertaining to the generation of the new list of cells to GenerateCells, then assign the return value to the newCells variable:

`var newCells = GenerateCells();`